js/script.js

```diff
@@ -1,8 +1,15 @@
 $(function () {
     $(document).scroll(function () {
-        var $nav = $(".fixed-top");
-        $nav.toggleClass('scrolled', $(this).scrollTop() > $nav.height());
-        var $text = $(".text-white");
-        $text.toggleClass('dark', $(this).scrollTop() > $nav.height())
+        var $nav = $("#nav-bar");
+        var $navSupportedContent = $("#navbarSupportedContent");
+        var isShown = $navSupportedContent.attr("class").includes(" show");
+        if(!isShown)
+            $nav.toggleClass("scrolled", $(this).scrollTop() > $nav.height());
+    });
+    $("#toggler").click(() => {
+        var $nav = $("#nav-bar");
+        if ($(this).scrollTop() < $nav.height()) {
```

Comment:
I don't think we need the `if statement` because the toggler will only show up on small screen devices, and we always want the navbar to turn dark regardless of the scroll height. Makes sense?

Reply:
no we don't want to toggle the class if it's already dark

Reply:
Why not, what difference does it make?

Reply:
because toggling at that time will make it like
transparent when it should be dark.,
and dark when it should be transparent.